src/slacko.ml

```diff
@@ -584,19 +590,23 @@ type history_result = [
   | timestamp_error
 ]
 
-let base_url = "https://slack.com/api/"
-
-let endpoint e =
-  base_url ^ e
-  |> Uri.of_string
-
 (* internal *)
+let default_base_url = "https://slack.com/api/"
+
 let optionally_add key value uri = match value with
   | None -> uri
   | Some value -> Uri.add_query_param' uri (key, value)
 
 let definitely_add key value = optionally_add key (Some value)
 
+let unauthed_endpoint ?(base_url=default_base_url) e =
```

Comment:
I'd give `e` a name like `method` since that's what Slack calls it in the documentation.